package.json

```diff
@@ -28,47 +28,47 @@
     "rebuildsass": "npm rebuild node-sass"
   },
   "dependencies": {
-    "@angular/common": "10.2.5",
-    "@angular/compiler": "10.2.5",
-    "@angular/core": "10.2.5",
-    "@angular/forms": "10.2.5",
+    "@angular/common": "13.3.11",
+    "@angular/compiler": "13.3.11",
+    "@angular/core": "13.3.11",
+    "@angular/forms": "13.3.11",
     "@angular/http": "*",
```

Comment:
I think `@angular/http` is deprecated and upgrading to v13 is good time to drop it